### DISCUSSION TITLE
what is necessary to create my own custom redstone wire?

### DISCUSSION THREAD

**Discussion by Endative** [FabricMC/fabric-api#2788] [Mod Dev Support]
**i wouldn't actually like it to power anything**, i'd just like to figure out how to get the properties of a redstone wire. help would be highly appreciated as i've been trying for a while now and all I've got is this:

![2022-12-24_14 33 15](https://user-images.githubusercontent.com/96704210/209442686-9f420f29-9993-4b9d-ba92-f0e2bb7c511b.png)

the hitbox is correct but there is no texture

**Comment by warjort** [comment]
Not sure what you are asking?

The blockstates model is this:
https://github.com/misode/mcmeta/blob/assets/assets/minecraft/blockstates/redstone_wire.json
you can find the block properties defined in `RedstoneWireBlock`
